CMS-Enterprise/mint-app
# [MINT-3201]/mto ready for review by new table

# MINT-3201

## Description

1. New MTO_INFO Table
    a. This follows a new pattern, the model_plan_id is the same as the primary key. 
    b. the model_plan_id is returned from the SQL script, but it is the same as the id. We could re-visit this, or just go back to standard paradigm for simplicity. 


## How to test this change

Please compare changes with https://github.com/CMS-Enterprise/mint-app/pull/1520. Instructions copied from there for ease of testing. Please also check if there are any areas that have been forgetten.
1.  Mark section ready for review by user
2. Ensure that it marked ready for review and has a time stamp
4. Change user (through postman variables)
5. Try to mark ready for review again
6. Ensure that it didn't change who marked ready for review or the time stamp
7. Set it as not ready for review
8. Ensure it no longer is ready for review
9. Mark ready for review again with the other user
10. Ensure the time and time stamp was marked

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
